cardano-cli/src/Cardano/CLI/EraBased/Options/Genesis.hs

```diff
@@ -273,18 +274,22 @@ pGenesisCreateTestNetData envCli =
     pSupply :: Parser (Maybe Lovelace)
     pSupply =
       Opt.optional $ fmap Lovelace $ Opt.option Opt.auto $ mconcat
-        [ Opt.long "supply"
+        [ Opt.long "total-supply"
         , Opt.metavar "LOVELACE"
-        , Opt.help "The initial coin supply in Lovelace which will be evenly distributed across initial, non-delegating stake holders. Defaults to 1 million Ada (i.e. 10^12 Lovelace)."
-        , Opt.value 1000000000000
+        , Opt.help $ unlines [ "The initial coin supply in Lovelace which is evenly distributed across stake holders. Defaults to 1 million Ada (i.e. 10^12 Lovelace)."
```

Comment:
```suggestion
        , Opt.help $ mconcat [ "The initial coin supply in Lovelace which is evenly distributed across stake holders. Defaults to 1 million Ada (i.e. 10^12 Lovelace)."
```
This seems to still be glued into a single line, so `unlines` seems confusing here.